sportsref/ncaaf/players.py

```diff
@@ -147,8 +152,17 @@ def rushing_and_receiving(self):
         :returns: Pandas DataFrame with stats.
         """
         doc = self.getDoc()
-        table = doc('#rushing')
+        table = doc('#all_rushing')
```

Comment:
So the reason I am using all_whatever is because just rushing or passing works fine for the top-most table, but I could not get the pyquery to find the tables with those ids on the lower portion of the player pages. For example, for Jabrill Peppers, the methods cannot see 'rushing' id. For some reason it gets lost after the enormous comments included on those lower tables. That's also why I included the:

if df.empty and table.length > 0:
 +            tree = etree.fromstring(str(table))
 +            comments = tree.xpath('//comment()')
 +            comment = etree.tostring(comments[0])
 +            contents = comment.replace("<!--", "").replace("-->", "")
 +            table = pq(contents)
 +            df = sportsref.utils.parseTable(table)